apps-rendering/src/grid/LeftCentreBorder.tsx

```diff
@@ -0,0 +1,36 @@
+// ----- Imports ----- //
```

Comment:
Is this going to be re-used in multiple places? Far as I can tell it's only used in the `ImmersiveLayout` and all you really need is the function that returns styles from here.

Reply:
Yes there are several layouts that include borders along the side (galleries, for example), so the expectation is that it'll be used for each of those. Also, as mentioned in the description, there may be a couple more of these border components for the other borders and the API might change a bit 🤔.

There's also a possibility this grid module might get lifted to Source, so I'm trying to design it as generically as possible to make it easy to lift out if and when that happens.